data/map beyond patir.txt

```diff
@@ -0,0 +1,203 @@
+	habitable 1400
+	belt 1074
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/black hole corona core"
+		period 1
+	object
+		sprite "star/black hole corona"
+		period 1
+	object "One-Way Wormhole Entrance"
+		distance 1510.2286
+		period 1
+		offset 35
+	object
+		sprite planet/wisp
+		distance 1510.2286
+		period 1
+		offset 35
+
+system Chanai
+	pos 9883.5 -34.5
+	haze _menu/haze-chanai
+	habitable 1250
+	belt 1887
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/black hole 3 core"
+		period 1
+	object
+		sprite "star/black hole 3"
+		period 1
+	object "Chanai Structure"
+		sprite "planet/station-chanai"
+			scale 0.5
+		distance 1541.4
+		period 9215.7
+
+system Ghila
+	pos 9820.5 -100.5
+	haze _menu/haze-ghila
+	habitable 1715
+	belt 1165
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/black-hole-still-core"
+		period 1
+	object
+		sprite "star/black-hole-still"
+		period 1
+	object
+		sprite "planet/penrose machine"
+			scale 0.5
+		period 1
+	object "One-Way Wormhole Exit"
+		distance 1626.8
+		period 10283.9
+	object
+		sprite planet/wisp
+		distance 1626.8
+		period 10283.9
+
+system Lathia
+	pos 10107.5 -332.5
+	haze _menu/haze-lathia
+	habitable 1715
+	belt 1815
+	fleet "Magic Asteroid" 250
+	object
+		sprite star/neutron
+		period 1
+	object
+		sprite planet/rock11
+		distance 1822.21
+		period 227.72
+	object
+		sprite planet/base
+		distance 2723.62
+		period 1372.93
+
+system Maithi
+	pos 10092.5 -489.5
+	haze _menu/haze-maithi
+	habitable 425.92
+	belt 1379
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/black hole 5 core"
+		period 1
+	object
+		sprite "star/black hole 5"
+		period 1
+	object "One-Way Wormhole Exit"
+		sprite "planet/wormhole-blue"
+		distance 4316.1251
+		period 10.1251
+		offset 180
+
+system Mitera
+	pos 9957.5 -92.5
+	haze _menu/haze-mitera
+	habitable 3645
+	belt 1464
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/black hole 4 core"
+		period 1
+	object
+		sprite "star/black hole 4"
+		period 1
+	object "One-Way Wormhole Entrance"
+		sprite "planet/wormhole-blue"
+		distance 4580.2286
+		period 11.3382
+		offset 180
+
+system Sarifa
+	pos 10136.5 -406.5
+	haze _menu/haze-sarifa
+	habitable 5486.68
+	belt 1757
+	fleet "Magic Asteroid" 900
+	object
+		sprite "star/f8"
+		distance 549
+		period 1
+		offset 78
+	object
+		sprite "star/black hole star core"
+		period 1
+	object
+		sprite "star/black hole star"
+		period 1
+
+system Thepa
+	pos 9975.5 -2.5
+	haze _menu/haze-maithi
+	habitable 320
+	belt 1793
+	object
+		sprite "star/black hole 6 core"
+		period 1
+	object
+		sprite "star/black hole 6"
+		period 1
+
+system Balnii
+	pos 10268.3 -359.3
+	haze _menu/haze-sarifa
+	arrival 1100
+	habitable 360
+	belt 1497
+	fleet "Magic Asteroid" 900
+	hazard "Balnii Black Hole Event Horizon" 1
+	hazard "Black Hole Gravity" 2
+	object
+		sprite "star/big black hole core"
+		period 1
+	object
+		sprite "star/big black hole"
+		period 1
+
+planet "One-Way Wormhole Entrance"
+	description ``
```

Comment:
What is this description for? NPCs land if there is a `spaceport`, but adding a description won't do anything.